soh/soh/GameMenuBar.cpp

```diff
@@ -402,6 +402,8 @@ namespace GameMenuBar {
                     UIWidgets::Tooltip("Explosions are now a static size, like in Majora's Mask and OoT3D. Makes bombchu hovering much easier.");
                     UIWidgets::PaddedEnhancementCheckbox("Prevent Bombchus Forcing First-Person", "gDisableFirstPersonChus", true, false);
                     UIWidgets::Tooltip("Prevent bombchus from forcing the camera into first-person mode when released.");
+                    UIWidgets::PaddedEnhancementCheckbox("Aiming reticle for the bow/slingshot", "gBowReticle", true, false);
+                    UIWidgets::Tooltip("Aiming with a bow or slingshot will display a reticle as with the hookshot.");
```

Comment:
It'd be great to have something in here mentioning that you must have the arrow nocked/slingshot pulled back (not sure the best way to phrase it) to see the reticle.